modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dto/v1_0/util/DDMFormValuesUtil.java

```diff
@@ -163,34 +163,29 @@ private static Locale _getDefaultLocale(
 	}
 
 	private static Map<String, List<ContentField>> _toContentFieldsMap(
-		List<ContentField> contentFields) {
+		ContentField[] contentFields) {
 
-		if (contentFields.isEmpty()) {
+		if (contentFields == null) {
 			return new HashMap<>();
```

Comment:
@JanisZhang I think you can use `Collections.emptyMap()` here..

Reply:
@dantewang Then it need not create a separate Map object for each call. Thanks